hw_7/task_34.py

```diff
@@ -0,0 +1,40 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в 
+# его стихах ритм. Поскольку разобраться в его кричалках не 
+# настолько просто, насколько легко он их придумывает, Вам стоит
+# написать программу. Винни-Пух считает, что ритм есть, если число 
+# слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то
+# они разделяются дефисами. Фразы отделяются друг от друга пробелами. 
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры. В ответе 
+# напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”,
+# если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам  
+
+phrases = input().split()
+vowels = {"а", "о", "у", "ы", "э", "е", "ё", "и", "ю", "я"}
```

Comment:
Тут можно просто использовать строку
```suggestion
vowels = "аяуюоеёэиы"
```

Reply:
а разве через сэт это не будет работать быстрее?

Reply:
Вы правы, проверка на наличие элемента  во множестве проходит с константной сложностью, против линейной в других типах, но для объекта, состоящего их 10 элементов, это настолько незначительно, что разницей можно пренебречь в угоду читаемости.